app/helpers/friendships_helper.rb

```diff
@@ -1,4 +1,3 @@
-# rubocop:disable Lint/EmptyConditionalBody
 module FriendshipsHelper
   def verify_friendship(friend_id)
     Friendship.where('(user_id = ? and friend_id = ?) OR (user_id = ? and friend_id = ?)',
```

Comment:
[OPTIONAL] You should note methods like this should be better done in your model. 
There should be NO application logic in the views nor helpers (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...). Everything connected with data processing should be encapsulated in models and used in controllers.